### DISCUSSION TITLE
Projects only displaying one issue (should be many)

### DISCUSSION THREAD

**Discussion by sarahjsquire** [community/community#67636] [Projects and Issues] [answered]
Github is investigating the issue ([githubstatus.com incident](https://www.githubstatus.com/incidents/zjchv3zvfg50))
Tickets still exist. You can access yours at [github.com/issues](https://github.com/issues) or by following [this tip in the discussion](https://github.com/orgs/community/discussions/67636#discussioncomment-7050926)

### Select Topic Area

Bug

### Feature Area

Projects

### Body

My organization uses projects. All issues disappeared from all projects except one issue per product. We noticed around 4:35pm ET. Example of an empty board:
<img width="1433" alt="Screenshot 2023-09-19 at 4 47 32 PM" src="https://github.com/community/community/assets/3231369/c1cd9964-eb4a-40e4-b36a-c2201ba7d8f8">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by danjger** [comment]
Seeing the same thing here. nothing in project but one issue is appearing
<img width="2042" alt="image" src="https://github.com/community/community/assets/984525/22d7dd46-710d-4ff3-b043-341abf0e6fbb">

**Comment by Torewin** [comment]
This just happened to me. All of the issues are no longer tagged to that Project - a fix is going to every label and adding it back. Looks like the data was just wiped.

**Comment by rcpeene** [comment]
same here

**Reply by maebius** [reply]
And same thing here.

**Comment by GregoryPost** [comment]
Same.

**Comment by young-guo-ai** [comment]
same for me as well, though filtering for `project:<project id>` in `https://github.com/<org>/<repo>/issues` still comes up with the correct issues, so looks like not all data is lost

**Reply by danjger** [reply]
yea. nice catch, so there is hope. I looked at the project attributes of an issue that was in the project and it was blank
<img width="624" alt="image" src="https://github.com/community/community/assets/984525/951a6ecf-3fc6-4664-ae1b-6433dd32d48f">

**Reply by ezheidtmann** [reply]
This workaround should be the accepted answer. Use the "Project" dropdown on the Issues list for a repository.

The limitations are lack of columns and you can only view the issues on one repo at a time.

**Reply by jvherck** [reply]
This workaround does not work for me. Right as I clicked the corresponding project it brought me to a 500 error page...

**Reply by ldhasson** [reply]
Work around doesn't work for me, plus, i believe that wouldn't cover cards that have not been converted to an actual issue.

**Reply by trungnhm1998** [reply]
This works for me but due to org the syntax will be `project:org_name/project_id`.

**Comment by Toumash** [comment]
Same

**Comment by rbm0407** [comment]
same

**Comment by KNSeger** [comment]
Seeing the same issue in our org.

**Comment by goldiegadde** [comment]
We are running into this issue as well, we have 10-12 different project boards and all the issues disappeared from the project board, the issue UX does not show anything

**Comment by dequejenn** [comment]
Same

**Comment by hc-dinarak** [comment]
Same here

**Comment by Maclenn77** [comment]
Same issue started happening today after I assigned one issue to a colaborator at 2:50pm (-6 UTC). 
![issue github](https://github.com/community/community/assets/1808402/0834a078-2d8c-4c9d-a225-d1559b3a618f)

**Comment by matthewwong525** [comment]
Same thing happened with me.

All my work gone :(

**Comment by michael-siek** [comment]
All tickets look complete ¯\_(ツ)_/¯

**Comment by nathan-aquila** [comment]
Same here

**Comment by danjger** [comment]
Issue appeared on github status page so something is happening officially now.
https://www.githubstatus.com/

**Comment by wise-king-sullyman** [comment]
Seeing the same in several of my orgs as well. At least it happened at the end of the day here on the East coast 😆

**Comment by dsharer** [comment]
Seeing the same thing in my org.. in all our projects, like 4 or 5 different ones

**Comment by danielbraghini** [comment]
We had more than two hundred activities on github project on the new model, but now, is empty

**Comment by TOoSmOotH** [comment]
Same.. noticing all issues do not have a project assigned any more.

**Comment by jjustu** [comment]
friendly LOTR side-eye, because, same. 
![mordor](https://github.com/community/community/assets/91919500/0d56e682-2371-4b83-bedf-a164bc27ee52)

**Comment by javiervalero** [comment]
It looks to me that all issues but one lost their associations with projects.
If that's the case is a much worse scenario than just a display issue.

**Comment by 5orenso** [comment]
Same on my projects. Only 1 issue loading. Total disaster!

**Comment by megazordy** [comment]
Everything gone and all activities deleted, and if it doesn't come back it will be a lot of work 🤦‍♂️

![image](https://github.com/community/community/assets/16438605/7d56a916-f9c2-4a03-b262-8edaabf29953)

**Comment by sarahLy9** [comment]
same

**Comment by MaiconDM** [comment]
The same thing happened in my projects. GitHub help me work. 🤷‍♂️🤷‍♂️🤷‍♂️🤷‍♂️🤷‍♂️

**Comment by wlaubernds** [comment]
ditto. we rely heavily on projects and labels to customize views. looking forward to hearing from github.

**Comment by germanotm** [comment]
Same. I'm not seeing the status either

**Comment by UrsaPolarisRecords** [comment]
I am experiencing the same issue with the new Projects boards. It would appear Classic project boards are unaffected.

**Comment by mikaylagarvey** [comment]
Upsetti-spaghetti PM here, would really love to hear that I don't have to do all of this over again by hand... 😭

**Comment by UrsaPolarisRecords** [comment]
PO here wondering if I need to manually go and re-triage the hundreds of issues on the project board before the sprint kickoff tomorrow 🤷

**Comment by juampaweb** [comment]
yess Same for all my issues in my projects :(

**Comment by misslame** [comment]
Same

**Comment by o0amandagomez0o** [comment]
Really hoping there is some type of version control on our projects as well, I see there's a history of the last change but can't seem to find where it's cached... >.<

**Comment by GuyWhoCode** [comment]
Same

**Comment by Darren52BT** [comment]
Same

**Comment by jmmrte** [comment]
Same

**Comment by holc-tkomp** [comment]
![image](https://github.com/community/community/assets/54799591/4a3a4494-bdb9-4a03-bba0-0e6893ec38fd)

**Comment by g1f** [comment]
All wanished

**Comment by ramiromagno** [comment]
same

**Comment by GussRw** [comment]
Same
![image](https://github.com/community/community/assets/9169125/25bd2876-f24d-4af1-900d-9f1a1039f354)

**Comment by Toumash** [comment]
Looks like we need to call for some *backup* : D

In the meantime lets re-watch this beauty 
https://www.youtube.com/watch?v=dsHyUgGMht0

**Comment by polfc21** [comment]
same

**Comment by neverboring1** [comment]
@ . @

**Comment by sarahLy9** [comment]
![image](https://github.com/community/community/assets/67919689/31516208-3572-4e21-b5ea-0c579e2d6189)

**Comment by MlanzAgea** [comment]
Sameee

**Comment by chrispybokchoy** [comment]
Same rip D:

**Comment by Cynub** [comment]
Same

**Comment by thenewseason** [comment]
✋ same here..

**Comment by Jokeren** [comment]
same here

**Comment by GabeLawFR** [comment]
Same here, the issued tickets can be found in the "issues" tab, but nothing left in my board!

**Comment by crankapps** [comment]
ya, it looks like they wiped the issues in the projects. my guess is they are hearing about this right about now.....

**Comment by hcnguyen1** [comment]
same

**Comment by freshcutdevelopment** [comment]
Yes, we're seeing the same in our Github org, all projects appear to have lost their relationship with issues.

**Comment by jvherck** [comment]
Same here. Out of nowhere all of them except 1 random task will disappear and i manually have to go through each issue again to add it to the project. This already happened multiple times. The issues are becoming just one big spam of messages saying "**USER** added this to **PROJECT**" and "**USER** moved this to **STATUS** in **PROJECT**". It's very frustrating!

**Comment by danjger** [comment]
its all BACK for me at least.

**Reply by jvherck** [reply]
For me it's still all gone. And top of that I'm getting 500 status errors everywhere...

<img width="960" alt="image" src="https://github.com/community/community/assets/75937214/1302de5d-45cb-42b7-85b4-c4aec35fedc7">

**Reply by Toumash** [reply]
Works just fine for me

**Reply by jvherck** [reply]
Error 500 page is gone now. But all my issues are still unlinked sadly.

Edit: 500 page is still here

**Reply by danjger** [reply]
all gone again.

**Comment by UrsaPolarisRecords** [comment]
Fixed for my Projects boards, looks like going to the "Projects" tab for an org still goes to 500 error page

edit: 1 minute later, projects tabs are working as usual and no longer returning a 500 error page.

**Reply by jvherck** [reply]
Error pages have been fixed for me too. However I still only have one issue on my board. Are all your issues back?

**Reply by UrsaPolarisRecords** [reply]
They were back but now they're down again

**Comment by jmmrte** [comment]
Looks like it's still a work in progress. Some issues are marked as project "None yet", but if I assign the project, I see `Content already exists in this project`. Time for a beer.

**Comment by GabeLawFR** [comment]
My tickets are back but I cannot write more, I get a 'oups something went wrong alert'

**Reply by jmmrte** [reply]
Same here:

<img width="349" alt="SCR-20230919-puuv" src="https://github.com/community/community/assets/55513810/4078e9de-4dc7-4d35-8005-cd303750547f">

**Comment by Leaversa** [comment]
same

**Comment by jessegavin** [comment]
Same

**Comment by jvherck** [comment]
all my issues are back on the board!

**Edit:** 10 seconds later it's all gone again and getting 500 errors again

**Comment by jmmrte** [comment]
[<img width="794" alt="SCR-20230919-pzal" src="https://github.com/community/community/assets/55513810/6ef2b265-532d-47c1-ae70-299f88c10488">](https://www.githubstatus.com/)

Check the latest status here: https://www.githubstatus.com/

**Comment by ldhasson** [comment]
Same thing here over All of our "new style" projects. This is quite insane! 

![image](https://github.com/community/community/assets/410503/38ae52fd-d86b-491d-8302-3e501d2a03b7)

**Comment by jvherck** [comment]
Everything is fixed for my Org. All issues are back and I can make changes to it.

**Reply by trungnhm1998** [reply]
Do your card attributes back too? Is any data lost?

**Reply by jvherck** [reply]
@trungnhm1998 everything is back. no data is lost!

**Comment by gustavohenriquefs** [comment]
![image](https://github.com/community/community/assets/87202278/76cfa8fa-40ae-422b-b2ec-f30a5017f06e)

There is only one task left (these others are the correct team to redo the work)

![image](https://github.com/community/community/assets/87202278/7719b6db-1623-41b5-8c94-857c09ec7d69)

**Comment by duo755** [comment]
I went into my repository and tried manually re-adding the issues to the project thinking I had somehow erased my project.  Now i'm afraid it won't fix :(

**Reply by adamlindquist** [reply]
Same, we are struggling to track down all the items that were previously in our project and even when we manually add them back in, we have to update all the project attributes again.

**Comment by trungnhm1998** [comment]
Same, still there 6 hours ago.

**Comment by NikolausDemmel** [comment]
Same in our project. The workaround does not work. These have not been converted to issues yet. Moreover, it's an org-level project, not a repository-level project, and I don't see a list of issues for the org.

**Reply by trungnhm1998** [reply]
The filter seems only to work if you have the correct syntax `project:org_name/project_id`.

https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-project-board

**Comment by MaximeDeus** [comment]
Hello,

Same here :(

**Comment by TasponK** [comment]
So this problem is come from Github or Misconfiguration ?

**Comment by philipmjohnson** [comment]
My issues have gone as well.

**Reply by philipmjohnson** [reply]
https://github.com/orgs/geogardenclub/projects/1

**Comment by PatrickBassan** [comment]
Just checked mine, same problem

**Comment by kalashnikovisme** [comment]
Thought that somebody deleted almost all issues. It was scary :slightly_smiling_face:

**Comment by rob-bateman** [comment]
same problem here, all of my boards are empty apart from issue #1 in each

**Comment by dwalt** [comment]
Lost all mine in two project dashboards but one project each. Projects were not linked to repo issues, so workaround doesn't work and seems I've lost all my data. Project (classic) dashboard appears to be fine.

**Reply by rob-bateman** [reply]
> Project (classic) dashboard appears to be fine.

haha yeah, feel stupid for upgrading now

**Comment by Baescott** [comment]
Same

**Comment by DNin01** [comment]
Looking at a project in an organization I'm part of, yep, everything disappeared, but I'm sure it's coming back as GitHub is [working on restoring **access to Projects data**](https://www.githubstatus.com/incidents/zjchv3zvfg50), implying that nothing was lost, but simply inaccessible.

> We continue working on restoring access to Projects data, along with an estimated time to full restoration.

**Comment by papamoamac** [comment]
and..... broken again. Someone keeps pressing the red button!!!!!   :-)

**Comment by lukemovement** [comment]
https://www.githubstatus.com/incidents/zjchv3zvfg50

A warning would have been nice instead of the brief panic this left me in thinking that I had accidentally deleted several months of planning :/ 

![image](https://github.com/community/community/assets/1868429/e8f5d3db-84a8-4288-804e-7a08931cdad7)

**Reply by papamoamac** [reply]
awesome thanks

**Comment by kumazawamt** [comment]
me too
![image](https://github.com/community/community/assets/43743137/3efa0842-a319-4af9-8048-04b05942c3e1)

**Comment by darrow97** [comment]
same here!

**Comment by jd-campbell** [comment]
Same here for all of my project lists.

**Comment by largek** [comment]
Same problem, all issues have had project associations removed, including all project items. This affects multiple projects using within repo issues management, as well as across-repo management for some projects. We also use projects in multiple organisations. This is potentially catastrophic, and points to a risky hole that needs securing in this part of the pipeline.

**Reply by adamlindquist** [reply]
Completely agree with one edit: replace 'potentially' with 'actually' . Figuring out where these items were is going to be many hours of manual investigation and reconstruction...or I go to bed and hope it's magically fixed by morning?

**Reply by mikaylagarvey** [reply]
110% - Thank goodness they fixed it. I legit rage quit the last 30min of my day, it was so frustrating to even begin thinking about 1,000's of items that needed prioritization, scheduling. . . ugh!! I love this tool so much but this type of scare doesn't bode well for my BP.

**Comment by kevin-mcm** [comment]
Same here 🤔

**Comment by johnfrazieratms** [comment]
Looks like they fixed it!  My projects are back.

**Comment by kumazawamt** [comment]
revival!

**Comment by mikaylagarvey** [comment]
BEAUTIFUL. Thank you so much, legit sweating bullets as a project ships in literally 2 days 😅

**Comment by Moealh** [comment]
same here! I thought I did something wrong

**Comment by aim-kawai** [comment]
same here!

**Comment by azenMatt** [comment, marked as answer]
The incident here has been resolved, as per the update in https://www.githubstatus.com/incidents/zjchv3zvfg50 we will make sure a root cause analysis by Sept 29th. 

Apologies for all the panic and extra work created for you all with this problem. We will keep you posted on how we make sure it doesn't happen again

**Reply by Superjd10** [reply]
I created some drafts and some issues last night for one of my org private projects, but logging in today to see what I had pending the draft disappeared and some issues were untagged...

**Reply by duo755** [reply]
Hey Matt, I'm seeing an issue with my project still. I get an error saying "This project failed to load. Sorry about that. Please try refreshing and contact us if the problem persists."  Could you point me in the right direction of where to reach out to get help?

**Reply by MlanzAgea** [reply]
The same thing happens to me, now im getting an error when I try to enter into one project

**Reply by azenMatt** [reply]
I've shared this with the team who are investigating

**Reply by duo755** [reply]
Appreciate it. To give some context for my situation, I was adding labels to my repo and when I saved a new one, I noticed the issue with projects. Thinking I had done something to remove all the issues from the project, I tried manually adding issues back to the project which may have caused this conflict I'm seeing now.

**Reply by azenMatt** [reply]
We have located a small number of projects that weren't fully recovered - the team are working on a fix at the moment

**Reply by azenMatt** [reply]
@duo755 @MlanzAgea - we believe we have resolved this now - can you check and see if you can now load the projects?

**Comment by keithm-phys** [comment]
Yesterday our team experienced the same issue as everyone else in this thread. Today though when we go to our project board we are getting the message:

"This project failed to load. Sorry about that. Please try refreshing and contact us if the problem persists."

<img width="867" alt="image" src="https://github.com/community/community/assets/137920749/5200c057-d606-4a8a-9cae-059f9aaef7fc">


We contacted Support and they told us that our issue has been passed on to the Engineering team and closed the ticket with us. 

Is there anything else we can do to get things working again?

**Reply by keithm-phys** [reply]
This is working for us now too! Thanks for the help and the quick resolution.

**Comment by duo755** [comment]
Thanks for the update. Fingers crossed my project is in there!

**Comment by duo755** [comment]
BINGO! Please thank the team, they've saved me a few hours of creating a
new one.

**Comment by steve-doc** [comment]
I'm still not seeing the issues I've signed to my project.  Is there a resolution please?

**Reply by azenMatt** [reply]
Are you able to share the URL to your project?

**Comment by steve-doc** [comment]
Hi there

Thanks for coming back.  This is the project.  Unfortunately empty now.

https://github.com/users/steve-doc/projects/4


Regards

Steve

**Reply by azenMatt** [reply]
We took a look into this and it appears to be unconnected to the incident. The project does have 5 items in it - all of which are in the archive. Perhaps they got there accidentally? Does that help?

**Comment by steve-doc** [comment]
I did look at the archive and saw a few things there. They were all locked and I couldn’t access them. Sent from my iPhone. Please excuse the brevity and any typos.On 11 Oct 2023, at 10:10, Matt Butler ***@***.***> wrote:﻿
We took a look into this and it appears to be unconnected to the incident. The project does have 5 items in it - all of which are in the archive. Perhaps they got there accidentally? Does that help?

**Comment by sethwhitaker** [comment]
I have a private repo that I have not developed for since 2020 that I came back too recently only to find all my issues are missing. This 2023 incident is the only thing that I could find that might be related. It looks like many people had their data restored, but it appears my repo issues list continues to be lost.  I cannot publicly share the repo, but if someone could reach out to me privately to assist that would be appreciated.

**Reply by azenMatt** [reply]
Hi @sethwhitaker - to help you get a response here - please reach out to GitHub support - https://support.github.com/contact - and someone there will be able to assist. Thanks

**Comment by VincyaneBadouard** [comment]
I have the same problem today, the content of my project has disappeared except for a few items
![github_bug](https://github.com/user-attachments/assets/aef02f9d-e09b-4550-9e2c-065dfed84879)

**Comment by conradwt** [comment]
I'm experiencing the same issue with several projects missing its data.

**Comment by jrdtechjourney** [comment]
I’m facing the same problem with my current project. Whenever I create an issue, it mysteriously disappears when I attempt to move it to the next column. 
<img width="1919" height="958" alt="Screenshot 2025-08-12 at 10 15 32 AM" src="https://github.com/user-attachments/assets/7eade660-dafe-4caf-a5c5-ce63d617d09d" />

**Reply by MaskerPRC** [reply]
me too

**Reply by cagrigider** [reply]
same for me too suddenly disappeared every task

**Reply by b0kja85** [reply]
I'm currently having this problem today.

**Reply by naypak98** [reply]
same problem

**Reply by Incrementis** [reply]
Same problem. Creating a draft issue is also deleted after reloading the page.

**Reply by jrdtechjourney** [reply]
> Same problem. Creating a draft issue is also deleted after reloading the page.

This behavior is precisely what I am experiencing on my end. I am able to create an edit to the issues in the repository, but they do not appear on the project board. However, if I create a new “user story,” it does appear on the repository issues but not on the project board.  So frustrating!

**Reply by dyeramosu** [reply]
Same issue here

**Reply by dyeramosu** [reply]
> Same problem. Creating a draft issue is also deleted after reloading the page.

I can see the issue in my repo but it doesn't show up in projects

**Reply by jrdtechjourney** [reply]
I believe it is working now!!!

**Reply by Incrementis** [reply]
Yes, it's working for me now, too. All previously created drafts are now included.

**Reply by MaskerPRC** [reply]
my`s now working

**Reply by chefdog** [reply]
I got the same issue, one day the project board all my tickets are gone. They do, however still show up, when you navigate to the repository and open the issues tab... anyone any solution?

**Reply by dschlaep** [reply]
Same issue for me, one day last week my project started showing only one issue out of several dozen other issues no longer show up in the projects view, though the issues themselves are still linked to the project

**Reply by Jianbe-03** [reply]
Same issue here currently, but they also don't appear anyway, I only see a single issue not more eventhough there are more issues created and that have their project set to that project.

**Reply by uldiskk** [reply]
About 4-5 hours ago, I started experiencing this in a new project, and then also in an old project. I have a free subscription on a private project and private repo.
